.changeset/pre.json

```diff
@@ -0,0 +1,19 @@
+{
+  "mode": "pre",
+  "tag": "alpha",
+  "initialVersions": {
```

Comment:
`initialVersions` here should be `graphiql`, `@graphiql/react`, `codemirror-graphql` and `@graphiql/toolkit` only. 

I imagine the changests CLI added all of these just because they had active diffs or changes, but there is no reason the language service, server, cli,  monaco-graphql, vscode-graphql, or any of these examples need to go into pre-release for these changes

Reply:
the dedicated branch is how things went south before, because then the main and dedicated branch (2.x at the time) got terribly out of sync!

that said, as long as we don't make any changes to the graphiql packages from main while this branch exists, and the plan is to merge it back into main, I think we will be fine? just rebase often!

Reply:
Yeah I also don't really favor this approach, but I don't see a better option 😕 (I also hope that we can get all of this done within the next couple of weeks, so this branch won't be around for a longer time 🤞 )

Then let's do it like this! I'll create a branch `graphiql-v2` where we'll merge all the breaking changes and the new design into. Once we reviewed and collected everthing we'll merge this branch into `main`. I'll also rebase the branch regularly so that it won't get out of sync and doesn't block other PRs being merged into `main` in the meantime.